mmpose/datasets/datasets/body3d/body3d_h36m_dataset.py

```diff
@@ -199,23 +190,31 @@ def evaluate(self, outputs, res_folder, metric='mpjpe', **kwargs):
         for _metric in metrics:
             if _metric == 'mpjpe':
                 _nv_tuples = self._report_mpjpe(kpts)
+            elif _metric == 'n-mpjpe':
+                _nv_tuples = self._report_mpjpe(kpts, align_pred_and_gt=False)
             else:
                 raise NotImplementedError
             name_value_tuples.extend(_nv_tuples)
 
         return OrderedDict(name_value_tuples)
 
-    def _report_mpjpe(self, keypoint_results):
+    def _report_mpjpe(self, keypoint_results, align_pred_and_gt=True):
         """Keypoint evaluation.
 
         Report mean per joint position error (MPJPE) and mean per joint
         position error after rigid alignment (MPJPE-PA)
+
+        Args:
+            align_pred_and_gt (bool): If true (default), the prediction and the
+                gt will be aligned by subtracting global position (root joint
```

Comment:
What does `align` mean here, "subtracting global position"?
By alignment, who will be subtracted? pred only, gt only, or both?
A clearer name would be better